### ISSUE TITLE
Mobile Price Range Classification

### ISSUE THREAD

**Issue by ayushi424** [prathimacode-hub/PyAlgo-Tree#82]
---
Title: Mobile Price Range Classification
About: Implementing step by step  Logistic Regression algorithm in python  with help of a project i.e. Mobile Price Range Classification to classify the target variable(price_range) with value of 0(low cost), 1(medium cost), 2(high cost) and 3(very high cost) .
Name: Ayushi Shrivastava
Label: Feature Request
Assignee: 'ayushi424'

---

Define You:

- [x] DevIncept Participant 
- [x] Contributor

**Is your feature request related to a problem? Please describe.**

I want to contribute step by step implementation of  **logistic regression algorithm** using a project as **Mobile Price Range Classification**  to classify the target variable(price_range) with value of 0(low cost), 1(medium cost), 2(high cost) and 3(very high cost) with the project structure as mentioned.

**Describe the solution you'd like...**

Implementing Logistic Regression Algorithm using a project as Mobile Price Range Classification with following step:

Data training using the mobile price classification dataset. Model creation which will include importing logistic regression from sklearn, initializing the classifier, then fit the training data into it and perform classification.

**Dataset to be used**
https://www.kaggle.com/iabhishekofficial/mobile-price-classification?select=train.csv

**Comment by prathimacode-hub** [comment]
Here we can add it using project name but have to specifically use only one algorithm. So change the details accordingly. @ayushi424

**Comment by ayushi424** [comment]
> Here we can add it using project name but have to specifically use only one algorithm. So change the details accordingly. @ayushi424

Changes done. Kindly check the details.
@prathimacode-hub

**Comment by prathimacode-hub** [comment]
Issue assigned. @ayushi424
